<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebuggerPythonPage</class>
 <widget class="QWidget" name="DebuggerPythonPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>449</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Python Debugger&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line11_2">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Python Interpreter for Debug Client</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="E5PathPicker" name="interpreterPicker" native="true">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Enter the path of the Python interpreter to be used by the debug client.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Debug Client Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="2">
       <widget class="E5PathPicker" name="debugClientPicker" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Enter the path of the Debug Client to be used.  Leave empty to use the default.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="standardButton">
        <property name="toolTip">
         <string>Select the standard debug client</string>
        </property>
        <property name="text">
         <string>Standard</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="customButton">
        <property name="toolTip">
         <string>Select the custom selected debug client</string>
        </property>
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Source association</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Enter the file extensions to be associated with the Python2 debugger separated by a space. They must not overlap with the ones for Python3.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sourceExtensionsEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="pyRedirectCheckBox">
     <property name="toolTip">
      <string>Select, to redirect stdin, stdout and stderr of the program being debugged to the eric6 IDE</string>
     </property>
     <property name="text">
      <string>Redirect stdin/stdout/stderr</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="pyNoEncodingCheckBox">
     <property name="toolTip">
      <string>Select to not set the debug client encoding</string>
     </property>
     <property name="text">
      <string>Don't set the encoding of the debug client</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>435</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>interpreterPicker</tabstop>
  <tabstop>standardButton</tabstop>
  <tabstop>customButton</tabstop>
  <tabstop>debugClientPicker</tabstop>
  <tabstop>sourceExtensionsEdit</tabstop>
  <tabstop>pyRedirectCheckBox</tabstop>
  <tabstop>pyNoEncodingCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>customButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>debugClientPicker</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
